src/app.py

```
@@ -25,16 +27,25 @@
 
 @app.route("/api/<title>", methods=["GET"])
 def get_title(title) -> str:
+    # ---
+    start_time = time.time()
     # ---
     if event is None:
         return jsonify({"error": "حدث خطأ أثناء تحميل المكتبة"})
     # ---
     json_result = event([title], tst_prnt_all=False) or {"result": ""}
     # ---
+    delta = time.time() - start_time
```

Ruff (0.8.2)

Local variable `delta` is assigned to but never used

Remove assignment to unused variable `delta`

(F841)